ceph-ansible-galaxy/build/build

```diff
@@ -1,10 +1,6 @@
 #!/bin/bash
 
-# the following method exists in scripts/build_utils.sh
-pkgs=( "ansible" )
-install_python_packages "pkgs[@]"
-
 cd "$WORKSPACE"/ceph-ansible
 # propagates the change in the necessary Ansible Galaxy repos.
 # i.e. https://github.com/ceph/ansible-ceph-common
-$VENV/ansible-playbook -i dummy-ansible-hosts contrib/splitup.yml --tags update 
+bash contrib/splitup.sh
```

Comment:
Thanks, so is it `"$WORKSPACE"/ceph-ansible/contrib/splitup.sh` or `$WORKSPACE"/contrib/splitup.sh`